tests/fixed/ControlStructures.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ControlStructures;
+
+use InvalidArgumentException;
+use Throwable;
+use const PHP_VERSION;
+
+class ControlStructures
+{
+    private const VERSION = PHP_VERSION;
+
+    /**
+     * @return iterable<int>
+     */
+    public function varAndIfNoSpaceBetween() : iterable
+    {
+        $var = 1;
+        if (self::VERSION === 0) {
+            yield 0;
+        }
+    }
+
+    /**
+     * @return iterable<int>
+     */
+    public function ifAndYieldSpaceBetween() : iterable
+    {
+        if (self::VERSION === 0) {
+            yield 0;
+        }
+
+        yield 1;
+    }
+
+    /**
+     * @return iterable<int>
+     */
+    public function ifAndYieldFromSpaceBetween() : iterable
+    {
+        if (self::VERSION === 0) {
+            yield 0;
+        }
+
+        yield from [];
+    }
+
+    public function ifAndThrowSpaceBetween() : void
+    {
+        if (self::VERSION === 0) {
+            return;
+        }
+
+        throw new InvalidArgumentException();
+    }
+
+    public function ifAndReturnSpaceBetween() : int
+    {
+        if (self::VERSION === 0) {
+            return 0;
+        }
+
+        return 1;
+    }
+
+    public function noSpaceAroundCase() : void
+    {
+        switch (self::VERSION) {
+            case 1:
+            case 2:
+                // do something
+                break;
+            case 3:
+                // do something else
+                break;
+            default:
+        }
+    }
+
+    public function spaceBellowBlocks() : void
```

Comment:
Nitpicking, but it's "below" (one "l") :)

Reply:
daym, thought so but IDE spell check did not complain, it was wtf moment for me, did not validate it tho. Thanks.